src/cascadia/WindowsTerminal/main.cpp

```diff
@@ -4,14 +4,50 @@
 #include "pch.h"
 #include "AppHost.h"
 
+#include <wil/result.h>

```

Comment:
Why isn't the one in LibraryIncludes.h good enough? Does it miss some sort of result type because it's included before some other header?